### DISCUSSION TITLE
Future plans for Drawer

### DISCUSSION THREAD

**Discussion by alexander-bobin** [reapit/elements#62] [General]
@willmcvay I understand that Drawer is a new component in Elements. Please let me know if that's not the case.

Are there any concrete future plans to add to the Drawer or provide any new components related to it?

**Comment by willmcvay** [comment]
@alexander-bobin We'll be led by the Design System on this. The Drawer was intended to be flexible enough to render any component within it so no obvious plans at the moment.
